docpages/example_programs/the_basics/using_callback_functions.md

```diff
@@ -0,0 +1,97 @@
+\page callback-functions Using Callback Functions
+
+When you create or get an object from Discord, you send the request to its API and in return you get either an error or the object you requested/created. Let's create a message and get it:
+
+~~~~~~~~~~~~~~{.cpp}
+#include <dpp/dpp.h>
+#include <fmt/format.h> // used to format text, install and define FMT_HEADER_ONLY to use
+
+struct GetMessage {
+    dpp::snowflake message_id = 1146725845457719367ULL;
+    dpp::snowflake channel_id = 1112789762521182211ULL;
+};
+
+int main() {
+    GetMessage gm;
+
+    std::string BOT_TOKEN = "Token Was here";
```

Comment:
Please don't do this, look at the other examples as to how we format  creating the bot.

Reply:
The string is global, GetMessage is no more